commands/trigger.go

```diff
@@ -92,13 +92,24 @@ var triggerFireCmd = &cobra.Command{
 			return werr
 		}
 
+		if resp.StatusCode == 204 {
+			fmt.Fprintf(color.Output,
+				wski18n.T("trigger /{{.namespace}}/{{.name}} did not fire as it is not associated with an active rule(s)\n",
+					map[string]interface{}{
+						"namespace": boldString(qualifiedName.GetNamespace()),
+						"name":      boldString(qualifiedName.GetEntityName())}))
+
+			return nil
```

Comment:
up for discussion... is this situation considered an error (i.e. non-zero exit code, stderr)?

Reply:
@mdeuser, I believe we agreed to return a non-zero exit code here.

Reply:
@dubee yeah.. that's what you and i agreed upon.  was looking if anyone had an opposite argument :-)

Reply:
which exit code would you return?